pages/docs/tutorials/native/using-intellij-idea.md

```diff
@@ -1,50 +1,176 @@
-select **Kotlin** on the left hand column and **Native | Gradle** on the right hand.
+   By default, your project will use Gradle with Kotlin DSL as the build system.
+   > Kotlin/Native doesn't support Maven and IntelliJ build systems.
+   {:.note}
 
-![Wizard Step One]({{ url_for('tutorial_img', filename='native/using-intellij-idea/wizard.png')}})
+4. Accept the default configuration on the next screen and click **Finish**.
 
-Click **Next** in the dialog and in the next step make sure **Automatically import this project on changes in build script** is checked. This is useful when starting
-out to make sure any immediate changes to the build script are imported automatically. 
+   ![Configure a native application]({{ url_for('tutorial_img', filename='native/using-intellij-idea/native-new-project-intellij-2.png') }})
 
-![Wizard Step Two]({{ url_for('tutorial_img', filename='native/using-intellij-idea/wizard-2.png')}})
+Your project opens. By default, the wizard creates the necessary `main.kt` file with the code that prints "Hello, Kotlin/Native!" to the standard output.
 
-On clicking **Next**, enter the path and name for the project.
+The `build.gradle.kts` file contains the project settings. Read more about the settings in the [Kotlin Multiplatform Gradle DSL reference](../../reference/mpp-dsl-reference.html).
 
-![Wizard Step Three]({{ url_for('tutorial_img', filename='native/using-intellij-idea/wizard-3.png')}})
+## Run the application
 
-This will complete the process and open the newly created project in the IDE. By default the wizard will create the necessary
-`Sample<TARGET>.kt` file and provide code for writing some string to the standard output. Note that `<TARGET>` varies based on the operating
-system the project was created on (Windows, Linux, macOS).
+Start the application by clicking **Run** next to the run configuration at the top of the screen.
 
-![Project]({{ url_for('tutorial_img', filename='native/using-intellij-idea/IDE-1.png')}})
+![Run the application]({{ url_for('tutorial_img', filename='native/using-intellij-idea/native-run-app.png') }})
 
-To run the project, simply invoke the [Run command in the IDE](https://www.jetbrains.com/help/idea/running-applications.html), by using the corresponding shortcuts or from the Run menu
+IntelliJ IDEA opens the **Run** tab and shows the output:
+![Application output]({{ url_for('tutorial_img', filename='native/using-intellij-idea/native-output-1.png') }})
 
-![Run]({{ url_for('tutorial_img', filename='native/using-intellij-idea/IDE-2.png')}})
+## Update the application
+
+### Count letters in your name
+
+1. Open the file `main.kt` in `src/<your_app_name>Main/kotlin`.
+
+   The `src` directory contains Kotlin source files and resources. The file `main.kt` includes sample code that prints "Hello, Kotlin/Native!" using the [`prinln()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/println.html) function.
```

Comment:
```suggestion
   The `src` directory contains Kotlin source files and resources. The file `main.kt` includes sample code that prints "Hello, Kotlin/Native!" using the [`println()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/println.html) function.
```

Reply:
Also please use relative links to the API reference pages.
(/api/latest/jvm/stdlib/.../...html)

Reply:
Fixed all links.